src/heitech.zer0mqXt.core/Patterns/RqRep.cs

```diff
@@ -128,7 +128,7 @@ internal RqRep(SocketConfiguration configuration)
             {
                 if (responderIsSetup)
                 {
-                    throw new ZeroMqXtSocketException("Responder for this instance of Socket exists. Use a new instance for each server");
+                    throw new ZeroMqXtSocketException("Responder for this instance of Socket exists. Use a new instance for each server!");
```

Comment:
```suggestion
                    throw new ZeroMqXtSocketException("Responder for this instance of Socket exists already. Use a new instance for each server!");
```

What do you think about adding "already" here?